ui/components/src/DateTimeRangePicker/AbsoluteTimePicker.tsx

```diff
@@ -29,6 +30,7 @@ interface AbsoluteTimeFormProps {
 export const AbsoluteTimePicker = ({ initialTimeRange, onChange }: AbsoluteTimeFormProps) => {
   const [timeRange, setTimeRange] = useState<AbsoluteTimeRange>(initialTimeRange);
   const [showStartCalendar, setShowStartCalendar] = useState<boolean>(true);
+  const timeZone = useTimeZone();
```

Comment:
This makes sense, however do you think its worth also providing the `formatWithTimeZone()` as injected utility as well?

```tsx
  const { timeZone, formatWithTimeZone } = useTimeZone();
  
  // the timezone (last parameter) is optional since it will take the injected context
  formatWithTimeZone(timeRange.start, DATE_TIME_FORMAT)
 ```

Reply:
Sure, will explore this approach